src/aura/CRLP_RollupsContainer/CRLP_RollupsContainerHelper.js

```diff
@@ -257,7 +257,15 @@
         var altText = cmp.get("v.labels." + type);
         var text = {message: message, title: title, alternativeText: altText};
         cmp.set("v.notificationText", text);
-        cmp.set("v.notificationClasses", "");
+        cmp.set("v.toastHideClass", "");
+
+        if (type === 'success') {
+            window.setTimeout(
+                $A.getCallback(function() {
+                    cmp.set("v.toastHideClass", "slds-hide");
```

Comment:
I thought about this, but since it's such a small one-liner and closeNotificationWindow is only in the controller right now, it seemed like a lot of work to have to move that into the helper just to use it in both places. I also thought about trying to use a transition hide, but the SLDS classes weren't cooperating (.slds-transition-hide did the same thing, and .transition-hide didn't hide it at all!).

Reply:
Makes sense!